plasma_framework/contracts/src/framework/registries/VaultRegistry.sol

```diff
@@ -1,11 +1,17 @@
 pragma solidity ^0.5.0;
 
 import "../utils/Operated.sol";
+import "../utils/Quarantine.sol";
+
+contract VaultRegistry is Operated, Quarantine {
+    uint8 public constant INITIAL_IMMUNE_VAULTS = 2;
```

Comment:
(I am not sure if my suggestion is better)
Does it worth to set this in `PlasmaFramework` instead? Just like `minExitPeriod` as different env/deployment might want to start with different num of vaults?

Reply:
btw, just see this today: https://ethereum.stackexchange.com/questions/3067/why-does-uint8-cost-more-gas-than-uint256

Surprisingly this might actually cost more (?)

Reply:
> (I am not sure if my suggestion is better)
> Does it worth to set this in `PlasmaFramework` instead? Just like `minExitPeriod` as different env/deployment might want to start with different num of vaults?

Yeah, I agree that it would be nice to have that as an option. But I suspect that PlasmaFramework will end up having a large amount of constructor options...

I'll make the change now anyway and we can re-think it later if it becomes too unwieldy

Reply:
yeah....also I am not really sure whether we need this or not. But yeah, let's just revisit if anyone complains XDD